src/cards/ha-persistent_notification-card.html

```diff
@@ -37,12 +37,60 @@
     stateObj: {
       type: Object,
     },
+
+    scriptLoaded: {
+      type: Boolean,
+      value: false,
+    },
+
+    loaded: {
+      type: Boolean,
+      computed: 'computeLoaded(scriptLoaded)',
+    },
   },
 
+  observers: [
+    'computeContent(stateObj, loaded)',
+  ],
+
   computeTitle: function (stateObj) {
     return stateObj.attributes.title || stateObj.entityDisplay;
   },
 
+  computeLoaded: function (scriptLoaded) {
+    return scriptLoaded;
+  },
+
+  loadScript: function () {
+    var success = function () {
+      this.scriptLoaded = true;
+    }.bind(this);
+
+    var error = function () {
+      console.error('Micromarkdown was not loaded.');
+    };
+
+    this.importHref(
+      '/static/micromarkdown/micromarkdown-js.html',
+      success, error
+    );
+  },
+
+  computeContent: function (stateObj, loaded) {
+    var el = '';
+    var message = '';
+    if (loaded) {
+      el = document.getElementById('pnContent');
```

Comment:
Polymer will make all DOM elements with an id available at `this.$`, so use `this.$.pnContent`. It's better that way because otherwise you would only be ever able to have 1 card with markdown syntax